src/project-details/project-details.js

```diff
@@ -78,6 +67,19 @@ export class ProjectDetails {
       }
 
       this.releases.forEach(r => { this.downloads += r.total_downloads; });
+
+      if (repo.badges && repo.badges.status) {
+        switch(repo.badges.status.toLowerCase()) {
+          case 'active':
+            this.badge_status_image = '/img/active_flame_final_28w_35h.svg';
+            break;
+          case 'inactive':
+            this.badge_status_image = '/img/inactive_zzz_final_32w_35h.svg';
+            break;
+          default:
+            this.badge_status_image = null;
+        }
```

Comment:
This should be changed so that the default image is 'img/pending_review_final_29w_35h.svg'.
This image is to be used when a repo has not been reviewed yet.